README.md

```diff
@@ -0,0 +1,37 @@
+# ElasticSearch on Rancher
+
+> Rancher configuration for running an Elasticsearch cluster
+
+## Services deployed:
+
+- `es-master`: Elasticsearch nodes, part of the Elasticsearch cluster, one runs on every host that has an `escluster=true` tag defined.
+- Justwatch [Exporter]: Exposes Prometheus metrics on the cluster status.
+- es-client: Rancher cattle DNS service, which points the `es-client` domain name to `es-master`, for backwards compatibility with existing services.
```

Comment:
Does the service name have to be `es-client` for the redirect to work. If not then a better name would be something like`es-dns-redirect`. I'm not proposing that this is done as part of this PR.

Reply:
The service name can be anything. I would suggest if we're going to change it, to remove the DNS alias entirely, and just use `es-master` instead.